src/state.rs

```diff
@@ -138,6 +278,26 @@ mod test {
         assert!(black_move.can_move_pseudo(b7, b6));
     }
 
+    #[test]
+    fn test_rook_moves() {
+        let board = test_simple_board_for_piece_lateral_king(Piece::Rook);
+        let a4 = Pos { rank: 3, file: 0 };
```

Comment:
If you wanted to get rid of these, an MVP would be to just make a constant for every position on a normal chess board.

Reply:
good point! bbd2df7786730ea00e99eef07d962986acde840f